apps/flux-system/automation/registry-credential-sync/base/kustomization.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-  - https://github.com/timja/flux2/manifests/integrations/registry-credentials-sync/azure?ref=kustomize-substitute-fixes
+  - https://github.com/fluxcd/flux2/manifests/integrations/registry-credentials-sync/azure?ref=3e8d1ae1d5e66709b53aacfa64e38bc70bc4ea9a
```

Comment:
locked to a known working commit, `main` branch which it was before is a bit scary, once it's tagged we could swap to that